code/modules/events/hivebot_invasion.dm

```diff
@@ -0,0 +1,54 @@
+/datum/round_event_control/hivebot_invasion //A hivebot swarm core impacts the station and warps in hivebots.
+	name = "Hivebot Invasion"
+	typepath = /datum/round_event/hivebot_invasion
+	weight = 2
```

Comment:
Weighting should be a little higher, considering Hivebots don't drop Syndi loot on death.